src/style.css

```
@@ -1 +1,79 @@
-/* Styles go here */
+:root {
+  --color-font: #000;
+  --color-active: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", 'Times New Roman', sans-serif;
+  color: var(--color-font);
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 1200px;
+  padding: 0;
+  margin: auto;
+}
+
+@media screen and (max-width: 1024px) {
+  .header {
+    width: 1024px;
+  }
+}
+
+.img {
+  display: block;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.nav__item:hover {
+  color: var(--color-active);
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  position: relative;
+  display: inline-block;
+  height: 60px;
+  line-height: 60px;
+  color: inherit;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+.is-active {
+  color: var(--color-active);
+}
+
+.is-active::after {
+  position: absolute;
+  content: "";
+  left: 50%;
+  bottom: 0;
+  transform: translateX(-50%);
+  width: 37px;
```

```suggestion
  width: 100%;
```
it is should be 100% because different links have different width